### ISSUE TITLE
Add test suite?

### ISSUE THREAD

**Issue by mandarg** [netsniff-ng/netsniff-ng#168]
I introduced a change in #166 that broke a use case that I knew existed in the back of my mind, but didn't test correctly. A test suite could have helped me catch it sooner.

From looking through the codebase, I don't think one currently exists  but I'd be interested in writing some up (for `mausezahn` at least). Any suggestions for frameworks, or pointers towards existing similar projects that have a good test suite? [bats](https://github.com/sstephenson/bats) and [Check](https://github.com/libcheck/check) are some things I have seen around that test at different levels.

**Comment by vkochan** [comment]
Right, there is no any testing code. For mausezahn/trafgen might be used scapy for sniffing and packets
comparing.
